libparsec/crates/client_connection/tests/sse.rs

```diff
@@ -0,0 +1,114 @@
+#![cfg(not(target_arch = "wasm32"))]
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+mod proxy;
+
+use futures_util::StreamExt;
+use libparsec_client_connection::{AuthenticatedCmds, ConnectionError, SSEResponseOrMissedEvents};
+use libparsec_platform_http_proxy::ProxyConfig;
+use libparsec_protocol::authenticated_cmds::latest as authenticated_cmds;
+use libparsec_testbed::TestbedEnv;
+use libparsec_tests_lite::{p_assert_eq, p_assert_matches, parsec_test};
+
+#[parsec_test(testbed = "coolorg", with_server)]
+#[timeout(std::time::Duration::from_secs(5))]
```

Comment:
Why this timeout ? is the test flacky ?

Reply:
> is the test flacky ?

I don't know :shrug:, only time will tell.

> Why this timeout ?

During development of the test, I've found out that if something goes wrong there will be no progress (i.e. the test will not finish).
This timeout is here to prevent the test for being blocked for too long (by aborting it).

Reply:
I think it's a good idea to have a timeout on tests.
What do you think of nextest's timeout system ? (https://nexte.st/book/slow-tests.html#terminating-tests-after-a-timeout)

In think it would be better to handle this with it so that we don't have to bother with marking each test with the timeout marker.
However from my quick look at this, it seems we must add a `.config/cargo.toml` file to configure the timeout, instead of just passing the config on the command line :cry:  (which would be very convenient given we want very small timeout value when debugging on local machine, and much larger value when running on the CI)

What do you think ?

Reply:
> What do you think of nextest's timeout system ?

That could work.

---

> it seems we must add a `.config/cargo.toml`

In the first paragraph of your link:

> add this to `.config/nextest.toml`

So no `.config/cargo.toml` involved :smile: 

---

> What do you think ?

A issue is needed

Reply:
I've created the issue #5625

Reply:
In the mean time, I've removed the rstest timeout